src/ape_networks/_cli.py

```diff
@@ -163,3 +203,44 @@ def _run(cli_ctx, provider: "SubprocessProvider"):
     else:
         provider.disconnect()
         cli_ctx.abort("Process already running.")
+
+
+@cli.command(short_help="Stop node processes")
+@ape_cli_context()
+@click.option("--pid", "process_ids", help="The PID of the process(es) to kill", multiple=True)
+@network_option(default=None)
+def kill(cli_ctx, process_ids, provider):
```

Comment:
I don't see where `provider` gets used (I am supposing that specifying the network option will only kill that one specific provider)

Reply:
[e5b468c](https://github.com/ApeWorX/ape/pull/2545/commits/e5b468c59e4b7ab4dd83d63e2d6335bc11bce660)